packages/verifiable-credentials/test/config.ts

```diff
@@ -72,3 +78,14 @@ export const connect = async (privateKey: string, customContexName?: string): Pr
 };
 
 
+export const getClientContext = async (uri: string): Promise<Context> => {
```

Comment:
This should be in a utils / helper file, not in the config file.

Same with `connect` method.

Reply:
the `getClientContext` and  `connectAccount` (connect) method have been moved to the utils file

Reply:
@cmcWebCode40 No, these need to be in a utils in verifiable-credentials. These are VC specific utilities, they shouldn't be in client-ts.